.editorconfig

```diff
@@ -0,0 +1,6 @@
+[*]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
+insert_final_newline = true
+end_of_line = lf
```

Comment:
Mind adding this file to .gitignore?

Reply:
Well I can, but I think it is a good thing to have around as it unities the code style. I know that for that there is `aegir lint` but this integrates directly with IDEs. You don't use it in other projects? I feel like I took it from some other JS package...
For example here: https://github.com/ipfs/js-ipfs/blob/master/.editorconfig